packages/web-frontend/src/containers/EnlargedDialog/EnlargedDialogContent.js

```diff
@@ -128,9 +128,10 @@ export class EnlargedDialogContent extends PureComponent {
   }
 
   renderBody() {
-    const { viewContent, errorMessage } = this.props;
+    const { viewContent, errorMessage, isLoading } = this.props;
     const noData = viewContent.data && viewContent.data.length === 0;
 
+    if (isLoading) return <LoadingIndicator />;
```

Comment:
Nice find. I'm not sure what it did before either but I think this is a good choice of component to use. I know there is no design but I think the transparent white background is probably overkill so could we have just the loading spinner with no background?

Reply:
@tcaiger Just updated this for you

Reply:
Nice - thanks